route/leastconnection_benchmark_test.go

```diff
@@ -37,9 +37,9 @@ func loadBalanceFor(strategy string, b *testing.B) {
 	var lb route.EndpointIterator
 	switch strategy {
 	case "round-robin":
-		lb = route.NewRoundRobin(pool, "")
+		lb = route.NewRoundRobin(pool, "", false, "meow-az")
 	case "least-connection":
-		lb = route.NewLeastConnection(pool, "")
+		lb = route.NewLeastConnection(pool, "", false, "meow-az")
 	default:
```

Comment:
We should probably have test for different AZs here

Reply:
added more detailed benchmarks.

Reply:
Resolving this conversation as the benchmark now contains tests for both algorithms, tests for favoring and not favoring the local availability zone, and two AZs.